distributions/Dist_yl4874.py

```diff
@@ -0,0 +1,24 @@
+#Yongning Lei (Intro to Experimental Phy Lab II) P = 2x
+
+import numpy as np
+from .base_distribution import BaseDistribution
+
+
+class Dist_yl4874(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
```

Comment:
the maximum of your distribution is 2. That's why your histogram gets cut off.

Reply:
sorry about that, i adjusted the f_max